static/code/svg.html

```diff
@@ -0,0 +1,71 @@
+<html>
+  <head>
+    <title>Minimal demo of SvgCanvas</title>
```

Comment:
The initial check-in of SVG-edit should probably have *only* included the third-party code.